tests/core/test_playback.py

```diff
@@ -185,6 +185,17 @@ def test_next_keeps_finished_track_in_tracklist(self):
 
         self.assertIn(tl_track, self.core.tracklist.tl_tracks)
 
+    def test_next_skips_over_unplayable_track(self):
+        tl_tracks = self.core.tracklist.get_tl_tracks()
+        self.audio.trigger_fake_playback_failure(tl_tracks[1].track.uri)
```

Comment:
Note that `trigger_fake_playback_failure()` causes the next `set_state` call in audio to return `False`. This is probably good enough for this, so just FYI.